internal/convert.go

```diff
@@ -179,6 +179,9 @@ func (conv *Conv) GetDDL(c ddl.Config) []string {
 	var ddl []string
 	for _, t := range tables {
 		ddl = append(ddl, conv.SpSchema[t].PrintCreateTable(c))
+		for _, i := range conv.SpSchema[t].Indexes {
```

Comment:
just a minor nit pick, but `index` would be more readable than `i` in this case

Reply:
updated from i to index